VKSample/VKSample/Services/PhotoService/PhotoService.swift

```diff
@@ -0,0 +1,108 @@
+// PhotoService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Cache images.
+final class PhotoService {
+    // MARK: - Private Constants.
+
+    private enum Constants {
+        static let separatorCharacter: Character = "/"
+        static let defaultNameString: Substring = "default"
+        static let imagesDirectoryString = "images"
+        static let defaultPlaceholderName = "userPlaceholder"
+    }
+
+    // MARK: - Public properties.
+
+    var placeholderImage: UIImage? = UIImage(named: Constants.defaultPlaceholderName)
+
+    // MARK: - Private properties.
+
+    private let container: DataReloadable
+    private var imagesMap: [String: UIImage] = [:]
+    private lazy var fileManager = FileManager.default
+
+    // MARK: - Initializers.
+
+    init(container: UITableViewController) {
```

Comment:
```suggestion
    init(container: UITableViewController) {
```
врятли сервис должен знать про таблицу и уметь с ней работать

Reply:
Решили оставить как есть